docs/_templates/custom-class-template.rst

```diff
@@ -0,0 +1,33 @@
+{{ fullname | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
+
+   {% block methods %}
+   .. automethod:: __init__
```

Comment:
You're right, we don't need `__init__`.  We've standardized it such that we place docs all in the class docstring rather than in the `__init__`.  I'll tweak the template.